CONTRIBUTING.md

```diff
@@ -2,29 +2,57 @@
 
 Operator Lifecycle Manager is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers, and other resources to help get contributions into operator-lifecycle-manager.
 
-# Email and Chat
+## Communication
 
-- Email: [operator-framework][operator_framework]  
+- Slack: [#olm-dev on the Kubernetes Slack](https://kubernetes.slack.com/archives/C0181L6JYQ2)
+- Google Group: [OLM Dev Google Group](https://groups.google.com/u/1/g/operator-framework-olm-dev)
+- Mailing List: [operator-framework-olm-dev@googlegroups.com](operator-framework-olm-dev@googlegroups.com)
```

Comment:
This link appears to be broken - it looks like it's linking to a local path in the directory. It wasn't immediately obvious to me where I would be able to find this link either.